index.js

```diff
@@ -4,18 +4,24 @@ import fetch from "./doppler.js";
 // For local testing
 if (process.env.NODE_ENV === "development" && process.env.DOPPLER_TOKEN) {
   process.env["INPUT_DOPPLER-TOKEN"] = process.env.DOPPLER_TOKEN;
+  process.env["INPUT_DOPPLER-PROJECT"] = process.env.DOPPLER_PROJECT;
+  process.env["INPUT_DOPPLER-CONFIG"] = process.env.DOPPLER_CONFIG;
 }
 
-const DOPPLER_META = [
-  "DOPPLER_PROJECT",
-  "DOPPLER_CONFIG",
-  "DOPPLER_ENVIRONMENT",
-];
+const DOPPLER_META = ["DOPPLER_PROJECT", "DOPPLER_CONFIG", "DOPPLER_ENVIRONMENT"];
 
 const DOPPLER_TOKEN = core.getInput("doppler-token", { required: true });
 core.setSecret(DOPPLER_TOKEN);
 
-const secrets = await fetch(DOPPLER_TOKEN);
+// Project and Config are required if a Personal token supplied
+const IS_PERSONAL_TOKEN = DOPPLER_TOKEN.indexOf("dp.pt") >= 0;
+if (IS_PERSONAL_TOKEN) {
+  console.log("[info]: Personal token detected. doppler-project and doppler-config inputs are now required.");
```

Comment:
Should we just change this to a check like `if (IS_PERSONAL_TOKEN && !(DOPPLER_PROJECT && DOPPLER_CONFIG)) { throw Error("project and config required...") }` or similar?

Reply:
Good call. Error output is much cleaner.

<img width="767" alt="CleanShot 2022-08-09 at 14 44 48@2x" src="https://user-images.githubusercontent.com/133014/183566369-88286d0f-f139-4111-833f-aea58c46dd25.png">